build/dnvm.sh

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Correct changes to this file have already been made. However the two `build.sh` changes remain relevant and haven't happened in this repo yet.